omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -103,9 +103,42 @@
 			</tr>
 		</table>
 	</fieldset>	
-	
 	<br/>
 	
+    <fieldset>
+        <legend>Provider Account</legend>
```

Comment:
@wyclif I thought you were asking for me to explain the code lol. You mean translate this to different languages yes?